docs/tasks/run-application/update-api-object-kubectl-patch.md

```diff
@@ -81,14 +78,14 @@ The output shows that the PodSpec in the Deployment has two Containers:
 
 ```shell
 containers:
-- image: nginx
-  imagePullPolicy: Always
-  name: patch-demo-ctr
-  ...
 - image: redis
   imagePullPolicy: Always
   name: patch-demo-ctr-2
   ...
+- image: nginx
```

Comment:
I found out that most recently created container will be positioned above the old one(s).